setup.py

```diff
@@ -1,15 +1,29 @@
 from setuptools import setup
 
-setup(name='cromwell-tools',
-      version='1.0.0.dev1',
-      description='Utilities for interacting with the Cromwell workflow engine',
-      url='http://github.com/broadinstitute/cromwell-tools',
-      author='Mint Team',
-      author_email='mintteam@broadinstitute.org',
-      license='BSD 3-clause "New" or "Revised" License',
-      packages=['cromwell_tools'],
-      install_requires=[
-          'requests'
-      ],
-      include_package_data=True
-      )
+CLASSIFIERS = [
+    "Development Status :: 4 - Beta",
+    "Natural Language :: English",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 2.7",
+    "Topic :: Scientific/Engineering :: Bio-Informatics",
+]
+
+
+setup(
+    name='cromwell-tools',
+    version='1.0.1',
+    description='Utilities for interacting with the Cromwell workflow engine',
+    classifiers=CLASSIFIERS,
+    url='http://github.com/broadinstitute/cromwell-tools',
+    author='Mint Team',
+    author_email='mintteam@broadinstitute.org',
+    license='BSD 3-clause "New" or "Revised" License',
+    packages=['cromwell_tools'],
+    install_requires=[
+        'requests',
```

Comment:
Extraneous comma

Reply:
It's optional according to pep8, but I like to keep any long-form lists like this with commas on the end (python is fine with it). In this case it's questionable if we want to keep the list in this form as we want to keep a small dependency number. From Guido: 

```
When trailing commas are redundant, they are often helpful when a version control 
system is used, when a list of values, arguments or imported items is expected to be 
extended over time. The pattern is to put each value (etc.) on a line by itself, always 
adding a trailing comma, and add the close parenthesis/bracket/brace on the next 
line. However it does not make sense to have a trailing comma on the same line as 
the closing delimiter (except in the above case of singleton tuples).

Yes:

FILES = [
    'setup.cfg',
    'tox.ini',
    ]
initialize(FILES,
           error=True,
           )
No:

FILES = ['setup.cfg', 'tox.ini',]
initialize(FILES, error=True,)
```

Reply:
OK, this makes sense to me then!